resip/recon/UserAgent.cxx

```diff
@@ -452,6 +459,23 @@ UserAgent::onSubscriptionNotify(SubscriptionHandle handle, const Data& notifyDat
    // Default implementation is to do nothing - application should override this
 }
 
+const char*
+UserAgent::sendMessage(const NameAddr& destination, const Data& msg, const Mime& mimeType)
+{
+   if(!mDum.getMasterProfile()->isMethodSupported(MESSAGE))
+   {
```

Comment:
if MESSAGE wasn't already in the MasterProfile, we wouldn't be able to receive MESSAGE requests.  Please log a warning message here when this scenario is detected.